tests/framework/installer/ceph_manifests.go

```diff
@@ -1868,7 +1868,7 @@ func (m *CephManifestsMaster) GetBlockPoolStorageClass(namespace string, poolNam
 }
 
 // GetFilesystem returns the manifest to create a Rook filesystem resource with the given config.
-func (m *CephManifestsMaster) GetFilesystem(namespace, name string, activeCount int) string {
+func (m *CephManifestsMaster) GetFilesystem(namespace, name string, activeCount int, activeStandbyCount int) string {
```

Comment:
Are you going to add this to the test yaml below so the new setting will be tested?

Reply:
Yes. I do intend to add.